dropgz/build/cniTest_windows.Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG ARCH
+ARG OS_VERSION
+FROM --platform=linux/${ARCH} mcr.microsoft.com/oss/go/microsoft/golang:1.20 AS azure-vnet
+ARG VERSION
+WORKDIR /azure-container-networking
+COPY . .
+RUN GOOS=windows CGO_ENABLED=0 go build -a -o azure-vnet.exe -trimpath -ldflags "-X main.version="$VERSION"" -gcflags="-dwarflocationlists=true" cni/network/plugin/main.go
+RUN GOOS=windows CGO_ENABLED=0 go build -a -o azure-vnet-telemetry.exe -trimpath -ldflags "-X main.version="$VERSION"" -gcflags="-dwarflocationlists=true" cni/telemetry/service/telemetrymain.go
+RUN GOOS=windows CGO_ENABLED=0 go build -a -o azure-vnet-ipam.exe -trimpath -ldflags "-X main.version="$VERSION"" -gcflags="-dwarflocationlists=true" cni/ipam/plugin/main.go
+
+FROM --platform=linux/${ARCH} mcr.microsoft.com/cbl-mariner/base/core:2.0 AS compressor
+ARG OS
+WORKDIR /dropgz
+COPY dropgz .
+COPY --from=azure-vnet /azure-container-networking/azure-vnet.exe pkg/embed/fs
+COPY --from=azure-vnet /azure-container-networking/azure-vnet-telemetry.exe pkg/embed/fs
+COPY --from=azure-vnet /azure-container-networking/azure-vnet-ipam.exe pkg/embed/fs
+RUN cd pkg/embed/fs/ && sha256sum * > sum.txt

```

Comment:
copy azure-vnet-telemetry.config as well

Reply:
last time i checked that config file is populated with default values and we don't need to copy it

Reply:
thats true but actual deployment copies it..so if there any regression in config (mostly it won't), then we may miss it. I like the installation to be same however we do in production